docs/blog/.authors.yml

```
@@ -7,3 +7,8 @@ jbemmel:
   name: Jeroen van Bemmel
   description: Webscale Architect
   avatar: https://avatars.githubusercontent.com/u/2031627
+
+sogawa:
+  name: Satoshi OGAWA
```

Hi @saogawa 
can I suggest to use the following way of writing the last name?
```suggestion
  name: Satoshi Ogawa
```